### PR TITLE
fix: fix a few typos and formatting errors

### PR DESCRIPTION
**Pull request by Eldemarkki**

### CODE REVIEW

File: pages/privacy.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default function Home({ copyrightYear }: InferGetServerSidePropsType<type
           <h2>5. Rekisterin tietosisältö</h2>
           <p>
             <IoLogoGithub />
-            <a href="https://github.com/Testausserveri/testausapis/blob/715fe1fa313327124909aa94c16bcc3b059911da/src/database/schemas/userInfo.js">Rekisterin tietokantarakenne on avointa lähdekoodia ja tarkasteltavissa myös GitHubissa</a>.
+            <a href="https://github.com/Testausserveri/testausapis/blob/main/src/database/schemas/userInfo.js">Rekisterin tietokantarakenne on avointa lähdekoodia ja tarkasteltavissa myös GitHubissa</a>.
```
Comment (Eldemarkki) [inline pages/privacy.tsx:89]: Should this instead be pinned to a single commit, in case that file is moved/deleted later?

Comment (Esinko) [reply pages/privacy.tsx:89]: I don't think so. If the file is moved and the link is incorrect, which probably means code has changed, the user should get a 404 and not incorrect information.

**Comment by Esinko** [comment]
Looks great overall. After all conversations are resolved, this should be merged asap.

**Review by Esinko — APPROVED** [review]

File: pages/privacy.tsx (hunk 1 above)
Comment (Eldemarkki) [reply pages/privacy.tsx:89]: Yes that's what I thought as well. Currently it linked to an old version so I changed it to that.

**Comment by Eldemarkki** [comment]
@Esinko this could probably be merged?

**Comment by Esinko** [comment]
Yup

**Pull request merged**